libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/schema/mysql/MySQLNoGreaterThan5740SchemaAccessor.java

```diff
@@ -164,10 +164,10 @@ public void switchDatabase(String schemaName) {
     @Override
     public List<DBObjectIdentity> listUsers() {
         MySQLSqlBuilder sb = new MySQLSqlBuilder();
-        sb.append("SELECT DISTINCT user FROM mysql.user");
+        sb.append("SELECT user,host FROM mysql.user");
```

Comment:
what is the behavior if there exist multile records for same user ?

Reply:
1. add DISTINCT for remove duplicate record if db user and host is same
2. web will show multi db user record with different host, and odc user can select any of them to be locked